tests/publiccloud/check_registercloudguest.pm

```diff
@@ -148,6 +148,52 @@ sub check_instance_unregistered {
     }
 }
 
+sub new_registration {
+    record_info('Starting registration...');
+    $instance->ssh_script_retry(cmd => "sudo ${path}/registercloudguest $regcode_param", timeout => 300, retry => 3, delay => 120);
+    check_instance_registered($instance);
+
+    record_info('Testing docker on the instance.');
+    $instance->ssh_assert_script_run("source /etc/profile.local");
+    $instance->ssh_assert_script_run("systemctl start docker.service");
+    record_info("systemctl status docker.service", script_output("systemctl status docker.service"));
```

Comment:
First value is the title while it does not have any technical limitation but this is text which openQA need to put in small square in the page so string longer then 5-10 chars will just not fit and will be cut in the middle . You may keep string which you wrote but play pay attention in VR how it will look like ...

Reply:
Indeed, e.g. shorter like "docker service", visible in a box in test details.